src/lib/interceptor/read-many-request.interceptor.ts

```diff
@@ -40,8 +40,11 @@ export function ReadManyRequestInterceptor(crudOptions: CrudOptions, factoryOpti
 
             const pagination = PaginationHelper.getPaginationRequest(paginationType, req.query);
 
+            const withDeleted = _.isBoolean(customReadManyRequestOptions?.softDeleted)
+                ? customReadManyRequestOptions.softDeleted
+                : crudOptions.routes?.[method]?.softDelete ?? CRUD_POLICY[method].default.softDeleted;
             const query = await (async () => {
-                if (PaginationHelper.isNextPage(pagination)) {
+                if (pagination.hasQuery()) {
```

Comment:
`isNextPage` and `hasQuery` have the same business logic within a function, but the

In this line, it is more meaningful to know if request is `not on the first page` than the purpose of having a query in the pagination.

Since these conditions can change, I don't think it's appropriate to change the method name.

Reply:
Initially, I also wanted to keep the original method name if possible. 
However, I found that AbstractPagination already has a property named `isNext`. - [code line](https://github.com/woowabros/nestjs-library-crud/blob/main/src/lib/abstract/abstract.pagination.ts#L76)
The `isNext` property seems to verify whether the query has been validly parsed and if there is a total count.

Having both the property isNext and method isNextPage existing simultaneously might be somewhat confusing, so I decided to change the name.
Would it be better to change the existing isNext property instead?

Reply:
I don't think `hasQuery is a better name than isNextPage` because the presence of a Query in Pagination can mean `a lot of different things`.

Also, I don't think it's a good idea to `complicate dependencies` by keeping Pagination's functionality open.

PaginationHelper.isNextPage simply means that the `current request is not the first page`, and such logic should not be of interest to the caller.

pagination.hasQuery means that a Query exists, and it is ambiguous what the state of not having a Query means.

Reply:
I got it. 
It seems that the dependencies might become unclear due to this change. 
I will revise the method by moving it back to PaginationHelper and submit a new PR. 

Thank you for the detailed review!